fortran/module_netcdf4_nc_interfaces.F90

```diff
@@ -408,13 +421,11 @@ Function nc_inq_compound_field_ndims(ncid, xtype, fieldid, ndims) BIND(C)
  End Function nc_inq_compound_field_ndims
 
 End Interface
-
-!------------------------------- nc_inq_compound_field_f ----------------------
+!------------------------------- nc_inq_compound_field ------------------------
 Interface
- Function nc_inq_compound_field_f(ncid, xtype, fieldid, name, offsetp, &
```

Comment:
I'm not sure what's going on with these name changes. Can you explain?

Reply:
The c-helper functions in fortran/nf_lib.c are not needed.  The netcdf-c functions are called directly.

Reply:
This is a good example of an important refactor which has nothing to do with -i8. Whether or not -i8 is accepted, I would like these changes. Can you submit them as a separate PR?